### ISSUE TITLE
Really appreciate this code, but can you incorporate the Store code too?

### ISSUE THREAD

**Issue by davie-robertson** [Protoss78/pwa-firebase-saga-kit#2]
This is the first time I have ever used Redux and I've been trying to figure out how to make the Stoe code work again (with or without sagas) - it would be really useful to have the Store code use as a real example of the thunk/saga change(s).

**Comment by Protoss78** [comment]
Yes. Will be shared soon. I'm currently working on a cloud firestore application and will integrate the relevant code in this sample. Should be up in a few days.
